templates/data/kafka/what-is-kafka.html

```diff
@@ -0,0 +1,566 @@
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              Kafka clusters can be scaled to increase processing capacity by adding additional servers, a huge improvement over the vertical scalability limitations of other message brokers.
+            </p>
+          </div>
+        </div>
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">Distributed processing</h3>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              Kafka workloads are distributed across many servers, which can massively improve performance by spreading concurrent client requests across the cluster.
+            </p>
+          </div>
+        </div>
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">Durable</h3>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              Kafka is designed to replicate copies of your event data across the cluster, so that data loss is minimised when a server goes offline.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Installing Kafka</h2>
+      </div>
+      <div class="col">
+        <p>
+          Kafka is a distributed system developed in Java, designed for computers running Ubuntu and other Linux distributions.
+        </p>
+        <div class="p-section--shallow">
+          <div class="p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/106d7c58-installing%20kafka.png",
+                      alt="",
+                      width="1200",
+                      height="676",
+                      hi_def=True,
+                      loading="lazy",
+                      attrs={"class": "p-image-container__image"},) | safe
+            }}
+          </div>
+        </div>
+        <p>
+          You can use Canonical's Charmed Kafka solution to deploy a fully supported Kafka service on cloud servers, Kubernetes, or even on your laptop.
+        </p>
+        <div class="p-cta-block u-no-padding--bottom">
+          <a class="p-button--positive" href="/data/docs/kafka/iaas">Try Charmed Kafka</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Kafka security</h2>
+      </div>
+      <div class="col">
+        <p>
+          Apache Kafka includes mature information security controls like TLS wire encryption, authentication and authorisation and access logging.
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50 p-section--shallow">
```

Comment:
Can you separate the structural classes into two different divs please for better coding practices

Reply:
Yes, thanks for catching!